tests/adapter/dbt/tests/adapter/basic/test_base.py

```diff
@@ -21,6 +22,7 @@ def models(self):
         return {
             "view_model.sql": base_view_sql,
             "table_model.sql": base_table_sql,
+            "table_model.py": base_table_py,
```

Comment:
I think we probably don't want to add this to the base adapter tests as not all adapters support running this

Reply:
Just the unittest above looks great to me

Reply:
All right, and it looks like you made that change, so you don't need anything from me?

Reply:
Just saw your other comment--looks good to me! Thank you!

Reply:
Sorry I just went ahead and did it thinking that it would be great for this to be included in the Beta release coming up soon!

Thank you so much for contributing this and the super quick response!!!